procurement/templates/procurement/includes/tender_filter.html

```diff
@@ -1,14 +1,14 @@
 {% load bootstrap4 %}
 
-            <fieldset class="col-sm-auto">
+            <fieldset class="d-flex align-items-end flex-wrap mb-md-2">
                 {% bootstrap_field filter.form.sort %}
                 {% bootstrap_field filter.form.state %}
```

Comment:
As mentioned in Slack, my instinct is to hide this form control entirely, and always just return "completed" tenders. The aim of the site is to expose contracts that are expiring – that requires the contract to already have been awarded, which is [represented as a `tender.status` of `"complete"`](https://standard.open-contracting.org/1.1/en/schema/codelists/#tender-status).

Reply:
@alexander-griffen Would it be helpful to have a chat about this today?